app/views/articles/index.html.erb

```diff
@@ -7,21 +7,25 @@
     <div class="col-lg-3"></div>
     <div class="col-lg-6">
       <% @articles.each do |article| %>
-        <div class="card mb-4">
-          <div class="card-body">
-            <div class="row justify-content-between">
-              <div class="col-4">
-                <div class="small text-muted"><%= article.created_at %></div>
-              </div>
-              <div class="col-4">
-                Status: <%= article.status %>
+        <%if article.user == current_user || article.status == 'public'%>
```

Comment:
Could we have a method in the controller that does this?

Reply:
Yes, to make the code view more aesthetically pleasing.